frontend/src/app/modules/social-page/filter-section/filter-section.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="filters-section-container">
+    <div class="btns-wrp">
+        <app-dropdown [isSingleChoice]="true" [btnLabel]="this.compatibilityBtnLabel" [resetEvent]="this.resetFiltersEvent" [data]="this.compatibilities" (selectedFilters)="update('compatibility', $event)"></app-dropdown>
+        <app-dropdown [isSingleChoice]="true" [btnLabel]="this.langBtnLabel" [resetEvent]="this.resetFiltersEvent" [data]="this.languages" (selectedFilters)="update('lang', $event)"></app-dropdown>
+        <app-dropdown [btnLabel]="this.levelBtnLabel" [resetEvent]="this.resetFiltersEvent" [data]="this.langLevels" (selectedFilters)="update('level', $event)"></app-dropdown>
+        <app-dropdown [btnLabel]="this.topicsBtnLabel" [resetEvent]="this.resetFiltersEvent" [data]="this.topics" (selectedFilters)="update('topic', $event)"></app-dropdown>
+    </div>
```

Comment:
We have separate components for language levels and interests, can you reuse them (maybe speak with Vladyslav about them)